src/core/features/filter/services/filter.ts

```diff
@@ -284,13 +284,15 @@ export class CoreFilterProvider {
         const data: CoreFiltersGetAvailableInContextWSParams = {
             contexts: contextsToSend,
         };
-        const preSets = {
+        const preSets: CoreSiteWSPreSets = {
             cacheKey: this.getAvailableInContextsCacheKey(contextsToSend),
             updateFrequency: CoreSite.FREQUENCY_RARELY,
             splitRequest: {
                 param: 'contexts',
                 maxLength: 300,
             },
+            // Use update in background, but always use the first value. If data is updated it will be stored in DB.
+            updateInBackground: true,
```

Comment:
```suggestion
            ...CoreSites.getReadingStrategyPreSets(CoreReadingStrategy.STALE_WHILE_REVALIDATE),
```

What do you think of doing this instead? I'm not sure why the settings related with caching are not set anyways.